code/weapon/weapon.h

```diff
@@ -209,6 +209,9 @@ typedef struct spawn_weapon_info
 
 #define MAX_SPAWN_TYPES_PER_WEAPON 5
 
+#define REARM_NUM_MISSILES_PER_BATCH 4              // default number of missiles rearmed per load sound
+#define REARM_NUM_BALLISTIC_PRIMARIES_PER_BATCH 100 // default number of bullets rearmed in per load sound
```

Comment:
These values are not needed in the header and could be moved to the source file.

Reply:
Totally missed that you requested this.  Is this a general principle I should follow?  If not needed by other source files, keep it out of the header?

Reply:
It's generally a good idea to keep the header as clean as possible. You will not always see that principle followed in the FSO code base but it's a good thing to do where possible.